01/ronaldokun/wordvalue.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+from data import DICTIONARY, LETTER_SCORES
+
+char = re.compile(r"[A-Z]{1}")  # I want to match individual UPPER CASE LETTERS
+
+
+def load_words() -> list:
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY, "r") as words:
+        return words.read().splitlines()
+
+
+def calc_word_value(word: str) -> int:
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum(LETTER_SCORES[c] for c in char.findall(word.upper()))
+
+
+def max_word_value(list_of_words: list = None) -> str:
```

Comment:
you say that `list_of_words` is of type `list` but in line 26 you cast the variable to a list. So there is either a wrong type or an unnecessary list cast

Reply:
At first I did that because in the test file was passed a tuple to test the function, so I put it this type cast to make it more general and cast iterables.